docx/blkcntnr.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, element, parent):
         super(BlockItemContainer, self).__init__(parent)
         self._element = element
 
-    def add_paragraph(self, text='', style=None):
+    def add_paragraph(self, text='', style=None, prev=None, ilvl=None):
```

Comment:
Not an urgent issue but it would be nice to extend docstring to explain new parameters.